pom.xml

```diff
@@ -142,14 +135,6 @@
                     <groupId>org.eclipse.jetty</groupId>
                     <artifactId>jetty-runner</artifactId>
                 </exclusion>
-                <exclusion>
```

Comment:
Also have you checked it works for some dummy Java project with HiveRunner as test dependency - I know I had a lot of problems with that, thats why I have excluded all of it.

Also this logging is a night mare..I am not sure this will work without these exclusions...;/